bin/o2a-run-sys-test

```diff
@@ -55,29 +60,27 @@ export COMPOSER_LOCATION=${COMPOSER_LOCATION:=europe-west1}
 COMPOSER_DAG_BUCKET=$(read_from_file COMPOSER_DAG_BUCKET)
 export COMPOSER_DAG_BUCKET=${COMPOSER_DAG_BUCKET:=""}
 
+COMPOSER_GKE_CLUSTER_NAME=""
+
+COMPOSER_WEB_UI_URL=""
+
+# Script global variables
 LOCAL_APP_NAME=$(read_from_file LOCAL_APP_NAME)
 export LOCAL_APP_NAME=${LOCAL_APP_NAME:=}
 
 PHASE=$(read_from_file PHASE)
 export PHASE=${PHASE:=convert}
 
-DATAPROC_USER=$(read_from_file DATAPROC_USER)
-export DATAPROC_USER=${DATAPROC_USER:=${USER:=example_user}}
-
 RENDER_AS_DOT="false"
 
-RUN_SSH_COMMAND="false"
+RUN_DATAPROC_SSH="false"
 
-RUN_ACCESS_SHELL="false"
+RUN_COMPOSER_SHELL="false"
```

Comment:
As above, I would exchange `SHELL` to `SSH` to obtain homogeneity.

Reply:
Shell and SSH are two different things.  SSH is a protocol enabling the safe execution of commands, but also copying files, port and socket forwarding and many others. Shell is just a shell. Allows you to execute commands.

Reply:
I think it really is a question of interpretation. If you look at https://en.wikipedia.org/wiki/Shell_(computing) - shell is really user interface (including graphical one). What we are actually doing is SSH-ing to the machine (and we are using particular shell for that). I'd also prefer to use SHELL as name.

Reply:
We have a specific kind of shell- command line shell. https://en.wikipedia.org/wiki/Command-line_interface#Command-line_interpreter

Almost every SSH gives access to the shell, and not every shell gives access to SSH.

Reply:
We discussed this topic offline.  We choose ``ssh``.